src/services/zipRecruiter.jsx

```diff
@@ -0,0 +1,18 @@
+const fetchData = async () => {
```

Comment:
Since this function is in the zipRecruiter file, it might be obvious here what data it's fetching, but when you use it anywhere else it might not be clear what data it's supposed to return, so it should probably be named something more descriptive.

Also, since this file is a service, there probably won't be any JSX in it, so it should use the `.js` file type and `.jsx`